### DISCUSSION TITLE
ویدیو آموزش اتصال به یوتیوب

### DISCUSSION THREAD

**Discussion by RtinLFS** [GFW-knocker/gfw_resist_HTTPS_proxy#18] [General]
با سلام و عرض ادب.
با تشکر از همه عزیزان مشارکت کننده و توسعه‌دهندگان اصلی.
این ویدیوی آموزشی رو ساختم و گفتم اینجا بفرستم اگر صلاح دونستید و مفید بود معرفی کنید. چون خیلی از عزیزان مخاطب حرفه‌ای نیستن و به مفاهیم این چنینی آشنایی ندارن و باید قدم به قدم براشون توضیح داد که چه کنن.

https://youtu.be/WwldPCQ7V4M

با سپاس از همه عزیزان

**Comment by GFW-knocker** [comment]
بسیار ممنون از زحمت شما
همچنان روی برخی نت ها لود ویدئو دچار مشکل هست
حتما بعد از stable شدن از رسانه شما استفاده خواهیم کرد.
